engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -551,6 +555,10 @@ public TutorialManager getTutMan() {
         return tutorialManager;
     }
 
+    public EntitySystemManager getEntitySystemManager() {
```

Comment:
What are the chances of injecting `Projectile` with the ESM so that we don't have to do this?